types/puppeteer/index.d.ts

```diff
@@ -1,17 +1,20 @@
-// Type definitions for puppeteer 1.12
+// Type definitions for puppeteer 1.15.0
```

Comment:
1.16

Reply:
Juste upgraded to 1.16.0
with 2 new methods.
@since XXXXX added in code.